packages/dart/sshnoports/bin/sshnp.dart

```diff
@@ -72,6 +72,17 @@ void main(List<String> args) async {
         throw e;
       });
 
+      // A listen progress listener for the CLI
+      // Will only log if verbose is false, since if verbose is true
+      // there will already be a boatload of log messages
+      logProgress(String s) {
+        if (!(params?.verbose ?? true)) {
+          stderr.writeln('${DateTime.now()} : $s');
+        }
+      }
+
+      sshnp.progressStream?.listen((s) => logProgress(s));
+
```

Comment:
Added a simple progress listener into the sshnp CLI so that, if run in non-verbose mode, the user will get enough output to keep them from wondering what is going on and why. (If run in verbose mode, a normal user gets too much information)